contrib/mesos/docs/issues.md

```diff
@@ -0,0 +1,67 @@
+## Known Issues
+
+### Pod Placement
+
+The initial plan was to implement pod placement (aka scheduling "constraints") using rules similar to those found in Marathon.
+Upon further consideration it has been decided that a greater alignment between the stock Kubernetes scheduler and kubernetes-mesos scheduler would benefit both projects, as well as end-users.
+Currently it is not possible to specify pod placement constraints for the kubernetes-mesos scheduler.
+This issue is being tracked here: https://github.com/mesosphere/kubernetes-mesos/issues/338
+
+### Resource Allocation
+
+Mesos is designed to handle resource accounting and enforcement across the cluster.
+Part of that enforcement involves "growing" and "shrinking" the pool of resources allocated for executor containers.
+The current implementation of the kubelet-executor launches pod-tasks as Docker containers (just like the upstream kubelet) and makes no attempt to actually "contain" the pods that are launched.
+Because the kubernetes-mesos scheduler cannot depend on the kubelet-executor to properly contain resources, it foregoes implementing accurate resource accounting.
+Instead CPU and memory resource limits are hardcoded into the scheduler for the time being.
```

Comment:
Everything in this paragraph is accurate, but I think you're "burying the lede" a bit. I would start this paragraph with "Resource requirements (limits) specified on Kubernetes Pods are currently ignored, both in the scheduler and on the node." (Then continue with everything you have here.)